users/README.md

```diff
@@ -34,16 +34,17 @@ number and email address.
 
 ## **Requests**
 
-|                         Route                          |             Description              |
-| :----------------------------------------------------: | :----------------------------------: |
-|                [GET /users](#get-users)                |   Returns all users in the system    |
-|           [GET /users/self](#get-usersSelf)            | Returns the logged in user's details |
-|  [GET /users/userId/:userId](#get-usersuseriduserid)   |    Returns user with given userId    |
-|       [GET /users/:username](#get-usersusername)       |   Returns user with given username   |
-|    [GET /users/:userId/badges](#get-usersidbadges)     | Returns badges assigned to the user  |
-|               [POST /users](#post-users)               |          Creates a new User          |
-|         [PATCH /users/self](#patch-usersself)          |       Updates data of the User       |
-| [PATCH /users/:id/temporary/data](#patch-usersidroles) |          Updates user roles          |
+|                         Route                               |             Description              |
+| :---------------------------------------------------------: | :----------------------------------: |
+|                [GET /users](#get-users)                     |   Returns all users in the system    |
+|           [GET /users/self](#get-usersSelf)                 | Returns the logged in user's details |
+|  [GET /users/userId/:userId](#get-usersuseriduserid)        |    Returns user with given userId    |
+|       [GET /users/:username](#get-usersusername)            |   Returns user with given username   |
+|    [GET /users/:userId/badges](#get-usersidbadges)          | Returns badges assigned to the user  |
+|               [POST /users](#post-users)                    |          Creates a new User          |
+|         [PATCH /users/self](#patch-usersself)               |       Updates data of the User       |
+| [PATCH /users/:id/temporary/data](#patch-usersidroles)      |          Updates user roles          |
+| [PATCH /users/update-archived](#patch-usersupdate-archived) |   Archive users if not in discord    |
```

Comment:
is this feature already merged? an update is an action I don't think should be a part of the API name.

Reply:
No, It's not merged yet. Can you suggest a name?

Reply:
`/users/archived` should be fine